pkg-manager/resolve-dependencies/src/updateLockfile.ts

```diff
@@ -120,7 +119,7 @@ function toLockfileDependency (
   if (opts.depPath[0] !== '/' && !pkg.id.endsWith(opts.depPath)) {
     result['id'] = pkg.id
   }
-  if (!isEmpty(pkg.peerDependencies ?? {})) {
+  if (Object.keys(pkg.peerDependencies ?? {}).length !== 0) {
```

Comment:
maybe just > 0?

it is 2 chars less

Reply:
updated the PR, do not really have preference on this